PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+#### What's this PR do?
+- Allows a customer to update their case
```

Comment:
Personally, I don't like leaving examples in the PR template, especially from other projects. It just means I have to delete a line every time I make a pull request